### PR TITLE
Fix fonts

### PR DESCRIPTION
**Pull request by dmorais-ledger**
These commits will fix fonts issues with OCR on Nano X & NanoS+

### CODE REVIEW

**Review by xchapron-ledger — COMMENTED** [review]
Can you fix the CI lint issue?

**Comment by xchapron-ledger** [comment]
Should we commit the `.inc` files? Only the `.json` ones are used.

File: speculos/mcu/ocr.py [hunk 1]
```diff
@@ -81,14 +84,51 @@ def display_char(font: bagl_font.Font, char: str) -> None:
...
+def get_json_font(json_name:str) -> Mapping[Char, BitMapChar]:
+    # If no json filename was provided, just return
+    if json_name is None:
+        return None
+
+    # Build the filename by adding api level information
+    json_name += f"-api-level-{OCR.api_level}.json"
+    # Now add the fonts path (JSON files are in speculos/fonts)
+    script_path = os.path.abspath(os.path.dirname(__file__))
+    json_name = os.path.join(script_path, "..", "fonts", json_name)
```
Comment (xchapron-ledger) [inline speculos/mcu/ocr.py:96]: You need to add speculos/fonts/*.json in MANIFEST.in, else that wqon't work in packaged version of speculos.

Comment (xchapron-ledger) [inline speculos/mcu/ocr.py:96]: You need to access the ressource following the cx_lib model: https://github.com/LedgerHQ/speculos/blob/master/speculos/main.py#L152

**Comment by dmorais-ledger** [comment]
> Can you fix the CI lint issue?

Sure, I'll do it. :)

**Comment by dmorais-ledger** [comment]
> Should we commit the `.inc` files? Only the `.json` ones are used.

For the same reason I added the .json files to the SDK in API_LEVEL_5 tag, I think it is better to provides the .inc files associated to the .json ones => in case of troubles, people will be able to compare these .inc files with the ones he's currently using in the SDK.

**Comment by xchapron-ledger** [comment]
> 

Why not just comparing the Json file with the one in the SDK then ?
And considering that the Json file is equivalent to the inc file in the SDK ? We should add a CI check for that!

**Pull request closed without merging**